packages/fx-core/src/common/templates.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import * as semver from "semver";
+
+export const templatesVersion = "0.1.*";
+export const tagPrefix = "templates@";
+export const preRelease = process.env.TEAMSFX_TEMPLATE_PRERELEASE || "";
```

Comment:
how about nightly build? I remember the conclusion is that the env variable is a flag about branch (main for nightly build, pre-release for pre-release and release with version for release)?

Reply:
I think, the conclusion is that for nightly, pre-release, we always have one release for each of them. So my decision is that we have 3 release called (0.0.0-nightly, 0.0.0-pre-release, 0.0.0-test). And we use env var TEAMSFX_TEMPLATE_PRERELEASE to specify which release we are using. I only create 0.0.0-test now, which is a test env for dev. (Notice, prerelase is the name of the 'xxx' of '0.0.0-xxx'. In semver, it is the term of the version part).